### ISSUE TITLE
When wallet has a nahmii balance in a currency not in the supported tokens list, getNahmiiBalances crashes

### ISSUE THREAD

**Issue by jonbern** [hubiinetwork/nahmii-sdk#74]
When a wallet has a nahmii balance in a currency that is not in the list of supported tokens, the `getNahmiiBalance` function will throw the error below:

```
TypeError: Cannot read property 'symbol' of undefined
    at Wallet.getNahmiiBalance (/usr/local/lib/node_modules/nahmii-cli/node_modules/nahmii-sdk/lib/wallet/wallet.js:85:36)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Error: Unable to retrieve the balance.
    at Object.handler (/usr/local/lib/node_modules/nahmii-cli/src/commands/show-commands/balance.js:21:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by jonbern** [comment]
It can happen if we remove a token from the supported token list, and if a wallet has an existing nahmii balance for that token.

How I discovered this issue was on our dev cluster, where the NII contract had been re-deployed and my wallet had an existing nahmii balance of the previous contract. This will likely not happen on a production deployment, but I think there can be cases where we want to remove tokens from the supported tokens list, and then this error condition can occur.

I'd say it's not a high priority to fix this, but ignoring the unsupported tokens and printing out an informative message about the token(s) that couldn't be found sounds like an appropriate fix to me.

**Comment by liamaharon** [comment]
Got it, thanks for the info. Will try to find time to address this this week.

**Comment by morfj** [comment]
I think just ignoring those "unknown" currencies would be good enough in most cases. The exception being settlement and withdrawals.

**Comment by morfj** [comment]
Fixed by #105
